### PR TITLE
configuration docs.

### PR DESCRIPTION
**Pull request by PawelPawelec-RDX**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/TransactionAssertionsOptions.cs [hunk 1]
```diff
@@ -1,119 +0,0 @@
...
-using FluentValidation;
-using Microsoft.Extensions.Configuration;
-using RadixDlt.NetworkGateway.Abstractions.Configuration;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Linq;
-
-namespace RadixDlt.NetworkGateway.DataAggregator.Configuration;
-
-public sealed record TransactionAssertionsOptions
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.DataAggregator/Configuration/TransactionAssertionsOptions.cs:74]: that wasn't used anywhere.

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs [hunk 2]
```diff
@@ -71,35 +71,12 @@ namespace RadixDlt.NetworkGateway.DataAggregator.Configuration;
 
 public sealed record LedgerConfirmationOptions
 {
-    /// <summary>
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:74]: these 3 weren't used anywhere.

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs [hunk 3]
```diff
@@ -71,35 +71,12 @@ namespace RadixDlt.NetworkGateway.DataAggregator.Configuration;
...
-    [ConfigurationKeyName("SufficientlySyncedStateVersionThreshold")]
-    public long SufficientlySyncedStateVersionThreshold { get; set; } = 1000;
-
     /// <summary>
     /// Gets or sets MaxCommitBatchSize.
     /// The maximum batch to send to the ledger extension service for committing.
     /// </summary>
     [ConfigurationKeyName("MaxCommitBatchSize")]
-    public long MaxCommitBatchSize { get; set; } = 300;
+    public long MaxCommitBatchSize { get; set; } = 1000;
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:79]: for some reason we had different value in json file, i see no point in having different value in these 2 files.

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs [hunk 4]
```diff
@@ -120,7 +97,7 @@ public sealed record LedgerConfirmationOptions
     /// or equal to MaxCommitBatchSize.
     /// </summary>
     [ConfigurationKeyName("LargeBatchSizeToAddDelay")]
-    public long LargeBatchSizeToAddDelay { get; set; } = 100;
+    public long LargeBatchSizeToAddDelay { get; set; } = 500;
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:100]: for some reason we had different value in json file, i see no point in having different value in these 2 files.

File: apps/DataAggregator/appsettings.json [hunk 5]
```diff
@@ -25,24 +25,5 @@
             "DisableCoreApiHttpsCertificateChecks": false,
             "CoreApiNodes": [],
         },
-        "Mempool": {
```
Comment (PawelPawelec-RDX) [inline apps/DataAggregator/appsettings.json:28]: removed as all of them are set by default to these values in in .cs files.

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs (hunk 3 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:79]: It's a breaking change. Please revert that.

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs (hunk 4 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:100]: ditto

**Review by krzlabrdx — CHANGES_REQUESTED** [review]

File: src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs (hunk 3 above)
Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs:79]: how is it breaking? it was set to 1000 in .json file and was effective for all environments. https://github.com/radixdlt/babylon-gateway/pull/756/files#diff-3ae0868be67e7c63004cb1768b3818f53803743a181be9833ed1322d5a0acaf6L38

**Review by krzlabrdx — APPROVED** [review]

**Pull request merged**